### PR TITLE
[VEN-2752]: Rewards distributor on zksync mainnet

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -4940,6 +4940,31 @@ export const globalConfig: NetworkConfig = {
...
+            asset: "XVS",
+            markets: ["WETH", "WBTC", "USDT", "USDC.e", "ZK"],
+            supplySpeeds: [
+              "370370370370370",
+              "370370370370370",
+              "277777777777777",
+              "555555555555555",
+              "277777777777777",
+            ],
+            borrowSpeeds: ["92592592592592", "92592592592592", "69444444444444", "138888888888888", "69444444444444"],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:4965]:
```suggestion
            borrowSpeeds: ["185185185185184", "185185185185184", "138888888888888", "277777777777776", "138888888888888"],
```

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -4940,6 +4940,31 @@ export const globalConfig: NetworkConfig = {
...
+          // WETH:    3840 XVS for Suppliers (80%)
+          //          960 XVS for Borrowers (20%)
+          // WBTC:    3840 XVS for Suppliers
+          //          960 XVS for Borrowers
+          // USDT:    2880 XVS for Suppliers
+          //          720 XVS for Borrowers
+          // USDC.e:  5760 XVS for Suppliers
+          //          1440 XVS for Borrowers
+          // ZK:      2880 XVS for Suppliers
+          //          720 XVS for Borrowers
```
Comment (chechu) [inline helpers/deploymentConfig.ts:4954]:
```suggestion
          // XVS Rewards Over 1 month (2,592,000 seconds)
          // WETH:    1920 XVS for Suppliers (80%)
          //          480 XVS for Borrowers (20%)
          // WBTC:    1920 XVS for Suppliers
          //          480 XVS for Borrowers
          // USDT:    1440 XVS for Suppliers
          //          360 XVS for Borrowers
          // USDC.e:  2880 XVS for Suppliers
          //          720 XVS for Borrowers
          // ZK:      1440 XVS for Suppliers
          //          360 XVS for Borrowers
```

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -4940,6 +4940,31 @@ export const globalConfig: NetworkConfig = {
...
+          {
+            asset: "XVS",
+            markets: ["WETH", "WBTC", "USDT", "USDC.e", "ZK"],
+            supplySpeeds: [
+              "370370370370370",
+              "370370370370370",
+              "277777777777777",
+              "555555555555555",
+              "277777777777777",
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply helpers/deploymentConfig.ts]: [aa8308a](https://github.com/VenusProtocol/isolated-pools/pull/441/commits/aa8308a3513061321dac51e784fd8a26c2478ac8)

File: helpers/deploymentConfig.ts (hunk 1 above)
Comment (GitGuru7) [reply helpers/deploymentConfig.ts]: [aa8308a](https://github.com/VenusProtocol/isolated-pools/pull/441/commits/aa8308a3513061321dac51e784fd8a26c2478ac8)

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (GitGuru7) [reply helpers/deploymentConfig.ts]: [aa8308a](https://github.com/VenusProtocol/isolated-pools/pull/441/commits/aa8308a3513061321dac51e784fd8a26c2478ac8)

**Review by chechu — APPROVED** [review]

**Pull request merged**